doc/lsc.txt

```diff
@@ -315,6 +315,14 @@ be smaller than with full syncs.
 By default the client will modify buffers in response to `workspace/applyEdit`
 messages. To disable edits set `g:lsc_enable_apply_edit` to `v:false`.
 
+                                                *lsc-snippet-support*
+                                                *g:lsc_enable_snippet_support*
+By default the client disables support for snippets as insert text. If set to
+`v:true` and supported by the server, insert text may contain snippets.
+
+Autocompletion of snippets requires a snippet plugin such as ultisnips or
```

Comment:
I think we should expand this. Something like "Autocompletion of snippets requires a snippet plugin which can complete using `user_data` on `CompleteDone`, such as neosnippet.vim"